core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -4259,21 +4255,14 @@ public void checkOrderBy(final boolean desc,
         .withHook(Hook.PLANNER, (Consumer<RelOptPlanner>) planner ->
             planner.removeRule(ENUMERABLE_MINUS_RULE))
         .explainContains(""
-            + "PLAN=EnumerableCalc(expr#0..3=[{inputs}], expr#4=[0], expr#5=[>($t2, $t4)], "
-            + "expr#6=[=($t3, $t4)], expr#7=[AND($t5, $t6)], proj#0..1=[{exprs}], "
-            + "$condition=[$t7])\n"
-            + "  EnumerableAggregate(group=[{0, 1}], agg#0=[COUNT() FILTER $2], agg#1=[COUNT() "
-            + "FILTER $3])\n"
-            + "    EnumerableCalc(expr#0..2=[{inputs}], expr#3=[0], expr#4=[=($t2, $t3)], "
-            + "expr#5=[1], expr#6=[=($t2, $t5)], proj#0..1=[{exprs}], $f2=[$t4], $f3=[$t6])\n"
+            + "PLAN=EnumerableCalc(expr#0..3=[{inputs}], expr#4=[0], expr#5=[>($t2, $t4)], expr#6=[=($t3, $t4)], expr#7=[AND($t5, $t6)], proj#0..1=[{exprs}], $condition=[$t7])\n"
```

Comment:
I agree that this formatting is better, but it makes it harder to diff
I think you should leave it this way, but perhaps you can comment on the PR when the plan is the same
or perhaps have two commits: one which shows the changes, and another one which reformats

Reply:
Reformatted.